sonar-db/src/main/java/org/sonar/db/user/RoleDao.java

```diff
@@ -61,7 +61,7 @@ private static int countResourceGroupRoles(DbSession session, Long resourceId) {
     return mapper(session).countResourceGroupRoles(resourceId);
   }
 
-  private static int countResourceUserRoles(DbSession session, Long resourceId) {
+  private static int countResourceUserRoles(DbSession session, long resourceId) {
```

Comment:
given how specific the changes are, you could be more specific in the commit's title